R/climate_match.R

```diff
@@ -653,7 +653,7 @@ climate_match <- function(region,
     }  
   }else{
     current_climate_map <- "maps were disabled"
```

Comment:
@SanderDevisscher: if maps are disabled, this three slots should be set to `NULL`. I like you want to inform the user about this. Let's do it using `message("maps are disabled")`.

Reply:
In the current version the map slots are filled with the "maps are disabled" string. 
Do you want me to add an additional warning message ?

Reply:
@damianooldoni ok

Reply:
done